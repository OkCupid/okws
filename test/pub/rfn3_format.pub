%{len(format_uint("%.2048x", 5))}
%{format_uint("%hhx", 512)}
%{format_uint("%llu", 5123445645)}
%{format_float("%1$.3f", 5.1234456)}
