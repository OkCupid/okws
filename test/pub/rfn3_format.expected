2048
0
5123445645
5.123
